File: Algo/moyen/Exo1.js
console.log(trouverLePlusLong("Il faudrait apprendre plus que les autres"));

function trouverLePlusLong(str) {
  words = str.split(" ");
  let longMax = 0;
  for (let i = 0; i < words.length; i++) {
     if (words[i].length > longMax){
         longMax = words[i].length;
     }
  }
  return longMax;
}

File: Algo/moyen/exo1.py
def trouverLePusLong(my_str):
    """function find the long word

    Args:
        my_str ([str]): [la phrase a analyse]

    Returns:
        [str]: [le mot le plus long ]
    """
    words = my_str.split()
    longMax = 0
    for word in words:
        if len(word) > longMax:
            longMax = len(word)

    return longMax


print(trouverLePusLong("Il faudrait apprendre plus que les autres"))